Sources/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -191,23 +191,32 @@ private extension PurchasedTransactionData {
 private extension PostReceiptDataOperation {
 
     func printReceiptData() {
-        do {
-            let receipt = try PurchasesReceiptParser.default.parse(from: self.postData.receiptData)
-            self.log(Strings.receipt.posting_receipt(
-                receipt,
+        switch self.postData.receipt.type {
+        case .jws:
+            self.log(Strings.receipt.posting_jws(
+                self.postData.receipt.serialized(),
```

Comment:
This can be simplified:
```suggestion
        case let .jws(content):
            self.log(Strings.receipt.posting_jws(
                content,
```